modular_skyrat/modules/customization/modules/mob/living/carbon/human/human.dm

```diff
@@ -103,4 +103,39 @@
 	to_chat(usr, span_notice("[try_hide_mutant_parts ? "You try and hide your mutant body parts under your clothes." : "You no longer try and hide your mutant body parts"]"))
 	update_mutant_bodyparts()
 
+/mob/living/carbon/human/verb/acting()
+	set category = "IC"
+	set name = "Feign Impairment"
+	set desc = "Pretend to be impaired for a defined duration."
+
+	if(stat != CONSCIOUS)
+		to_chat(usr, span_warning("You can't do this right now..."))
+		return
+
+	var/static/list/choices = list("drunkenness", "stuttering", "jittering", "stupidness")
+	var/impairment = tgui_input_list(src, "Select an impairment to perform:", "Impairments", choices)
+	if(!impairment)
+		return
 
+	var/duration = tgui_input_number(src, "How long would you like to feign [impairment] for?", "Duration in seconds", 25, 36000)
+	switch(impairment)
+		if("drunkenness")
+			SEND_SIGNAL(usr, COMSIG_ADD_MOOD_EVENT, "drunk", /datum/mood_event/drunk)
+			set_timed_status_effect(duration SECONDS, /datum/status_effect/speech/slurring/drunk, only_if_higher = TRUE)
+		if("stuttering")
+			set_timed_status_effect(duration SECONDS, /datum/status_effect/speech/stutter, only_if_higher = TRUE)
+		if("jittering")
+			set_timed_status_effect(duration SECONDS, /datum/status_effect/jitter, only_if_higher = TRUE)
+		if("stupidness")
+			set_timed_status_effect(duration SECONDS, /datum/status_effect/speech/stutter/derpspeech, only_if_higher = TRUE)
```

Comment:
not hugely a fan of this
```
	message = replacetext(message, " am ", " ")
	message = replacetext(message, " is ", " ")
	message = replacetext(message, " are ", " ")
	message = replacetext(message, "you", "u")
	message = replacetext(message, "help", "halp")
	message = replacetext(message, "grief", "grife")
	message = replacetext(message, "space", "spess")
	message = replacetext(message, "carp", "crap")
	message = replacetext(message, "reason", "raisin")
```

Reply:
Its one of the mild traumas you can get when you hit your head, called /datum/brain_trauma/mild/dumbness. this part of the datum is most relevant:
```
if(prob(capitalize_prob))
		var/exclamation = pick("!", "!!", "!!!")
		message = uppertext(message)
		message += "[apply_speech(exclamation, exclamation)]"
```
I understand that some of those mentioned replacements are a bit cringe though...

Reply:
Yeahhhh. If you want to adjust it to be a bit less uh. `"you" -> "u"`. Then that would be good. The exclamation stuff is fine enough, but it kind of generally just feels like something you can do ICly? Whereas stuff like drunkness is more complex and as a result harder, and obviously jittering is not realistically possible.

Reply:
Actually, let me see if I can find the correct datum for the brain trauma which scrambles word placement in a sentence, and then use that for stupidness instead.

Reply:
that would be preferable yeah